prefect_databricks/flows.py

```diff
@@ -306,18 +339,72 @@ async def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_pa
                 f"encountered an internal error: {jobs_runs_state_message}.",
             )
         else:
-            logger.info(
-                "Databricks Jobs Runs Submit (%s ID %s) has %s state. "
-                "Waiting for %i seconds.",
-                run_name,
-                multi_task_jobs_runs_id,
-                jobs_runs_life_cycle_state.lower(),
-                poll_frequency_seconds,
-            )
+            logger.info("Waiting for %s seconds.", poll_frequency_seconds)
             await asyncio.sleep(poll_frequency_seconds)
             seconds_waited_for_run_completion += poll_frequency_seconds
 
     raise DatabricksJobRunTimedOut(
         f"Max wait time of {max_wait_seconds} seconds exceeded while waiting "
         f"for job run ({run_name} ID {multi_task_jobs_runs_id})"
     )
+
+
+def log_state(array: Dict, run_id, state: Dict, run_page_url, logger, is_task=True):
```

Comment:
Can we add type annotations for all of these and specify the Dict key/value types as well?

Also perhaps this could be a private method?

Reply:
Done